pkg/lang/golang/arguments.go

```diff
@@ -12,8 +12,8 @@ type Argument struct {
 	Type    string
 }
 
-// GetArguements is passed a tree-sitter node, which is of type argument_list, and returns a list of in order Arguments
-func getArguements(args *sitter.Node) (arguments []Argument, found bool) {
+// GetArguments is passed a tree-sitter node, which is of type argument_list, and returns a list of in order Arguments
```

Comment:
I believe godoc is case sensitive, so this should match the function's name. It was a problem before, but since this is changing it, might as well fix it as well.